### ISSUE TITLE
[Navigation Bar] BottomNavigationView casts a shadow

### ISSUE THREAD

**Issue by manabu-nakamura** [material-components/material-components-android#2990]
**Description:** Do we have to use the BottomNavigationView with the LinearLayout?
https://m3.material.io/components/navigation-bar/implementation
https://github.com/material-components/material-components-android/issues/2953#issuecomment-1249034716
The BottomNavigationView **with** the LinearLayout casts **no** shadow:
```
<androidx.coordinatorlayout.widget.CoordinatorLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:layout_width="match_parent"
    android:layout_height="match_parent">
    <LinearLayout
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:layout_gravity="bottom">
        <com.google.android.material.bottomnavigation.BottomNavigationView
            android:id="@+id/bottomNavigationView"
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            app:menu="@menu/main" />
    </LinearLayout>
</androidx.coordinatorlayout.widget.CoordinatorLayout>
```
![Screenshot_20220922_130226](https://user-images.githubusercontent.com/71050561/191658307-98cad2ec-0711-4771-9527-6a38fed79ab0.png)
The BottomNavigationView **without** the LinearLayout casts **a pale** shadow:
```
<androidx.coordinatorlayout.widget.CoordinatorLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:layout_width="match_parent"
    android:layout_height="match_parent">
    <com.google.android.material.bottomnavigation.BottomNavigationView
        android:id="@+id/bottomNavigationView"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:layout_gravity="bottom"
        app:menu="@menu/main" />
</androidx.coordinatorlayout.widget.CoordinatorLayout>
```
![Screenshot_20220922_130204](https://user-images.githubusercontent.com/71050561/191658448-8a2ddb90-f14a-4155-a640-8a6607b736a6.png)
The catalog app uses the BottomNavigationView with the LinearLayout or the FrameLayout. If you remove the container, the BottomNavigationView will cast a pale shadow.
https://github.com/material-components/material-components-android/blob/master/catalog/java/io/material/catalog/bottomnav/BottomNavigationDemoFragment.java#L247:
```
//    inflateBottomNavs(layoutInflater, view.findViewById(R.id.bottom_navs));
    inflateBottomNavs(layoutInflater, (ViewGroup) view);
```
https://github.com/material-components/material-components-android/blob/master/catalog/java/io/material/catalog/bottomnav/res/layout/cat_bottom_nav_fragment.xml#L108-L112:
```
  <!--FrameLayout
      android:layout_width="match_parent"
      android:layout_height="wrap_content"
      android:id="@+id/bottom_navs">
  </FrameLayout-->
```
https://github.com/material-components/material-components-android/blob/master/catalog/java/io/material/catalog/bottomnav/res/layout/cat_bottom_navs.xml:
```
<!--LinearLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    android:layout_gravity="bottom"
    android:orientation="vertical">
  <com.google.android.material.bottomnavigation.BottomNavigationView
      android:id="@+id/cat_bottom_nav2"
      android:layout_width="match_parent"
      android:layout_height="wrap_content"
      app:menu="@menu/bottom_nav_menu"/>
</LinearLayout-->

<com.google.android.material.bottomnavigation.BottomNavigationView
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:id="@+id/cat_bottom_nav2"
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    android:layout_gravity="bottom"
    app:menu="@menu/bottom_nav_menu"/>
```
https://github.com/material-components/material-components-android/blob/master/catalog/java/io/material/catalog/bottomnav/res/layout/cat_bottom_navs_animated.xml:
```
<!--LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
  xmlns:app="http://schemas.android.com/apk/res-auto"
  android:layout_width="match_parent"
  android:layout_height="wrap_content"
  android:layout_gravity="bottom"
  android:orientation="vertical">

  <com.google.android.material.bottomnavigation.BottomNavigationView
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    app:menu="@menu/bottom_nav_animated_menu" />
</LinearLayout-->

<com.google.android.material.bottomnavigation.BottomNavigationView
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    android:layout_gravity="bottom"
    app:menu="@menu/bottom_nav_animated_menu"/>
```
https://m3.material.io/components/navigation-bar/overview: Elevation: No shadow

See also: https://github.com/material-components/material-components-android/issues/2953, https://github.com/material-components/material-components-android/commit/1322e61c4ca4229f07cc9b1ad0af6042a459d80c

**Expected behavior:** The BottomNavigationView **without** the LinearLayout casts **no** shadow.

**Minimal sample app repro:** [Editor2.zip](https://github.com/material-components/material-components-android/files/9622268/Editor2.zip)

**Android API version:** 33

**Material Library version:** 1.7.0-rc01

**Device:** Emulator 31.3.10

**Comment by Mika-89** [comment]
It's not bug. The main problem is that the bottom menu does not have any top margin. I think that's the reason why you dont see the elevation.

**Comment by imhappi** [comment]
fixed in e3937483c36449f71bc00a5b9376198345060f89
